sig-architecture/production-readiness.md

```diff
@@ -0,0 +1,51 @@
+# Production Readiness Review Process
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable, can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production.
+
+## Status
+
+The process and questoinnaire are currently under development as part of the
+[PRR KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/20190731-production-readiness-review-process.md), with a target that reviews will be needed for features
+going into 1.18.
+
+During the 1.17 cycle, the PRR team will be piloting the questionnaire and other
+aspects of the process.
+
+## Questionnaire
+
+* Feature enablement and rollback
+  - How can this feature be enabled / disabled in a live cluster?
+  - Can the feature be disabled once it has been enabled (i.e., can we roll
+    back the enablement)?
+  - Will enabling / disabling the feature require downtime for the control
+    plane?
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node?
+  - What happens if a cluster with this feature enabled is rolled back? What
+    happens if it is subsequently upgraded again?
+  - Are there tests for this?
```

Comment:
Clarify "this".  I suspect you mean, "are there tests for a disable, enable, disable, enable cycle", but you could also mean "upgrade, downgrade, upgrade" which seems pretty onerous at the moment.